src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
```

Comment:
no need for div, pass key to the MovieCard

Reply:
I don't understand why this is the case here, also. Because I need <div> with className 'card', this layout was by default... from MovieCard component I return fragment especially for this wrapping <div>.

Reply:
Если `<div className="card">` является неотъемлемой частью карточки то он должен быть в файле `MovieCard`
Тут останется только `<MovieCard {...movie} key={movie.imdbId} />`

Reply:
тут уже понял, логическая связь в итоге терялась (: